cfme/services/myservice/ssui.py

```diff
@@ -329,12 +324,15 @@ def step(self):
 
 @navigator.register(MyService, 'Retire')
 class MyServiceRetire(SSUINavigateStep):
-    VIEW = RetireServiceView
+    # TODO - remove check when BZ 1653288 is fixed
+    @property
+    def VIEW(self): #noqa
```

Comment:
@izapolsk in 5.10 confirmation pop-up window for retiring the service is not appearing so service is retiring without confirmation.
In 5.9 confirmation pop-up window appear after clicking on retire.
I added check. 
``` view.retire.click()``` is part of confirmation popup and for 5.10 its not needed as i said confirmation pop-up window is missing.

```    if self.appliance.version < "5.10":
              view.retire.click()
````

Reply:
```
self = VersionPick({Version('lowest'): <class 'cfme.services.myservice.ssui.RetireServiceView'>, '5.10': <class 'cfme.services.myservice.ssui.MyServicesView'>})
o = <cfme.services.myservice.ssui.MyServiceRetire object at 0x7fdd3d933310>, type = <class 'cfme.services.myservice.ssui.MyServiceRetire'>

    def __get__(self, o, type=None):
        if o is None:
            # On a class, therefore not resolving
            return self
    
>       result = self.pick(o.browser.product_version)
E       AttributeError: 'MyServiceRetire' object has no attribute 'browser'

../../../../cfme_venv/lib/python2.7/site-packages/widgetastic/utils.py:312: AttributeError
```

I think it should be VersionPicker not VersionPick. since here in mycase we don't have `browser` to navigation class `MyServiceRetire`, @izapolsk  please confirm.

Reply:
@izapolsk please review I have added VersionPicker and its Locally passing